examples/README.md

```diff
@@ -0,0 +1,61 @@
+﻿# Common switches
+
+OpenThread allows you to [configure](https://openthread.io/guides/build#configuration) the stack for different functionality and behavior. This configuration is based on changing compile-time constants during the build process using common switches listed in `/examples/common-switches.mk`.
```

Comment:
Nit: I would probably note somewhere that all the switches default to 0 unless otherwise noted in the description.